controller.go

```diff
@@ -868,8 +868,9 @@ func (c *controller) NewSandbox(containerID string, options ...SandboxOption) (s
 		if s.containerID == containerID {
 			// If not a stub, then we already have a complete sandbox.
 			if !s.isStub {
+				err = types.ForbiddenErrorf("container %s is already present: %v", containerID, s)
 				c.Unlock()
-				return nil, types.ForbiddenErrorf("container %s is already present: %v", containerID, s)
```

Comment:
Given this is a very unlikely error and given `sandbox` does not implement the `String()` method, there is no much value in printing the whole raw structure. I am thinking we can avoid the issue and just change the error format to 
`(container %s is already present in sandbox %s", containerID, s.ID())`